frontend_tests/node_tests/hotkey.js

```diff
@@ -418,4 +425,10 @@ run_test('motion_keys', () => {
     assert_mapping('down_arrow', 'drafts.drafts_handle_events');
     overlays.is_active = return_false;
     overlays.drafts_open = return_false;
+
+    hotkey.processing_text = return_true;
+    compose_state.composing = return_true;
+    assert_mapping('narrow_compose', 'narrow.to_compose_target', false, true, false);
+    compose_state.composing = return_false;
+    hotkey.processing_text = return_false;
```

Comment:
If we're not already doing so, we should test the case where we weren't composing.